runtime-api/bonded-currency/src/lib.rs

```diff
@@ -34,14 +34,18 @@ pub struct Coefficient<BitType> {
 	bits: BitType,
 }
 
+pub trait OperationValue<Balance> {
```

Comment:
Ideally, the `Balance` would be an associated type instead of a generic, but it's a very small detail.

Reply:
[19e12aa](https://github.com/KILTprotocol/kilt-node/pull/828/commits/19e12aaa4f4e07fff92c31f4aeaa497029688b04)